index.html

```diff
@@ -51,9 +51,9 @@
         </div>
         <section class="text-container">
           <div class="intro">
-            I'm Jim
+            <h1>I'm Jim
             <br />
-            Glad to see you!
+            Glad to see you!</h1>
           </div>
           <p class="desc">
             I'm a Software Developer! I can help you build a product, feature or
```

Comment:
- Provide hover effects to your buttons. With the requested design guidelines in Figma (color and fonts).